docs/src/dedicated-gen-3/overview.md

```diff
@@ -0,0 +1,55 @@
+---
+
+{{< description >}}
+
+Projects will often require a Dedicated production cluster when they require high availability, or they need more resources than offered by Professional. 
+Data location requirements is also common in choosing a Dedicated cluster, such as the need to deploy to a location Platform.sh does not currently have a region, or simply that production data cannot be kept on shared infrastructure.
+
+Dedicated Generation 3 works nearly identically to Grid environments, and does not require additional configuration on your part. The only difference is that of service availability.
+
+## Why a Generation 3?
+
+Our existing [dedicated infrastructure](/dedicated/overview) ensures increased uptime and availability for your applications and services, but configuration or application topology changes have to go through our Customer Success team, as the VMs are provisioned by us. Generation 3 gives you both the high availability from our Dedicated offer combined with the self-service flexibility and features of the Grid. 
+
+This means that you are able to edit your configuration yourself the same as you would on the grid - in your `services.yaml`, `routes.yaml`, and `.platform.app.yaml` files - and then see those changes reflected in your Dedicated Generation 3 environments every push without opening a ticket. 
+
+## Features
+
+* A minimum of three virtual machine cluster is associated with your default (production) environment, and also optionally with a staging environment. 
+* Configuration changes on development environments (through your `services.yaml`, `routes.yaml`, and `.platform.app.yaml` files) are reflected on these Dedicated Gen 3 clusters when you merge them. It is not necessary to open a support ticket to change production infrastructure like it is on Dedicated. 
+* You can deploy your application in any of our [supported cloud providers](/development/faq.md#which-geographic-zones-does-platformsh-cover).
+
+## Differences from the Grid
+
+Although Dedication Generation 3 adds plenty of features to your existing Grid applications, there are a few differences and limitations to be aware of when considering an upgrade. 
+
+### Available services
+
+In our [services documentation](/configuration/services) you will notice that not every service is available on Dedicated Generation 3 that is available on the Grid, nor is every version. The table below shows the currently available services and their versions for Dedicated Generation 3. 
+
+{{< gen-3-services >}}
+
+### Local mounts
+
+Because you get a redundant infrastructure, note that local mounts will be local to each Virtual Machine. Since you can't know which VM is going to handle a specific request, you also don't have a guarantee regarding which local mount is going to be used. Whether you actually want to use a local mount or in fact need to set up a network storage mount depends on your specific use-case.
+
+## Setting up
+
+### Migrating from the Grid
```

Comment:
The use case needs  to be clarified here, there are two paths: 1. Users that are on an enterprise offering on the grid. 2. Users that are pro users on the grid.